cmd/geth/config.go

```diff
@@ -183,6 +183,11 @@ func makeFullNode(ctx *cli.Context) *node.Node {
 		cfg.Eth.OverrideStoryNostoi = &v
 	}
 
+	if ctx.IsSet(utils.Override4844Flag.Name) {
```

Comment:
Probably better to just completely disable blob tx without having an option to enable it. We can enable later by changing the code. We don't want nodes to have different behaviors

Reply:
if possible, reject it earlier

Reply:
removed the flags and overrides